src/components/date-picker/date-picker.stories.ts

```diff
@@ -121,17 +121,24 @@ export const range = (): string =>
     )}
   </div>`;
 
+const selectRangeString = html`
+  <script>
+    let weekDays = document.querySelector(".week-days");
+    weekDays.querySelector("calcite-date-picker-day:first-child").selected = true;
+    weekDays.querySelector("calcite-date-picker-day:last-child").selected = true;
+  </script>
```

Comment:
Yeah unfortunately this won't work. Any JS needs to go into a [`play` function](https://storybook.js.org/docs/react/writing-stories/play-function) which don't work OOTB with custom elements. I will have some time to look into workarounds after v1. Can you add a `TODO` comment about adding a selectedRange story In lieu of this?

Reply:
> Can you add a TODO comment about adding a selectedRange story In lieu of this?

Can we create a follow-up issue instead? It'll give it more visibility and we can track it.

Reply:
So I should remove this test and create a follow up to to add to add the test later when the play function is implemented?